<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>683</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(164, 31, 19);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FAF5F1;</string>
    </property>
    <property name="text">
     <string>Welcome,</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_welcome_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FAF5F1;</string>
    </property>
    <property name="text">
     <string>Age: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblage">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="discount">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>APPLY DISCOUNT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="item1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>BURGER - 120</string>
    </property>
   </widget>
   <widget class="QPushButton" name="item2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>ICED TEA - 60</string>
    </property>
   </widget>
   <widget class="QPushButton" name="item3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>ICE CREAM - 45</string>
    </property>
   </widget>
   <widget class="QPushButton" name="item4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>FRIES - 70</string>
    </property>
   </widget>
   <widget class="QPushButton" name="item5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>SPAGHETTI - 145</string>
    </property>
   </widget>
   <widget class="QPushButton" name="item6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>7</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>FRIED CHICKEN - 160</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="itemtable">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>50</y>
      <width>361</width>
      <height>501</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>361</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>361</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>ITEM DESCRIPTION</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PRICE</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>570</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>REMOVE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_welcome_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>TOTAL : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pay">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 245, 241);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="paybtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>550</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>PAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string>LOGOUT</string>
    </property>
   </widget>
   <widget class="QLabel" name="fname">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FAF5F1;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lname">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FAF5F1;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="age">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 245, 241);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
